recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
+{% set sha256 = "ebc4e2bf8c8266212e342ff8ec4f6ae469e8c41a05d099b6778de8424ce32563" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - Makefile.patch  # [osx]
+    - build-python.patch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+
+  run:
```

Comment:
I am confused. This is exactly what I did with my PR [here](https://github.com/conda-forge/staged-recipes/pull/2628), but for osx only since `clang` doesn't have OpenMP. 

I guess this procedure is more stable for linux than osx?